This is only needed for kernel < 2.6.29 and conflicts with kernel 4.4.42

--- a/backport-include/linux/cred.h
+++ /dev/null
@@ -1,10 +0,0 @@
-#ifndef __BACKPORT_LINUX_CRED_H
-#define __BACKPORT_LINUX_CRED_H
-#include_next <linux/cred.h>
-#include <linux/version.h>
-
-#ifndef current_user_ns
-#define current_user_ns()	(current->nsproxy->user_ns)
-#endif
-
-#endif /* __BACKPORT_LINUX_CRED_H */
